index.js

```diff
@@ -1,7 +1,8 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
+// let counter = 1
 // console.log("%cQuestion 1", "color: red") 
-
+// let counter = 2
```

Comment:
Once you declare a variable (`let counter`), you can't redeclare it.

```javascript
let counter = 1 // declaration & assignment ✅

counter = 2 // re-assignment ✅

let counter = 3 //🚫  will produce an error because we are attempting to redeclare (`let counter`) a variable was already declared (on the first line here) 🚫
```